views/partials/nhl_player_card

```diff
@@ -0,0 +1,41 @@
+
+<script id="card" type="text/x-handlebars-template">
```

Comment:
that is WAY too meta for me.

On Wed, Apr 8, 2015 at 5:09 PM, Blake Parkinson <notifications@github.com>
wrote:

> In views/partials/nhl_player_card
> <https://github.com/blakeparkinson/sports2/pull/243#discussion_r28026300>:
>
> > @@ -0,0 +1,41 @@
> > +
> > +<script id="card" type="text/x-handlebars-template">
>
> the only thing that will change are whatever the stats are called right?
> We should just do a a sub sub template in this sub template
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blakeparkinson/sports2/pull/243/files#r28026300>.
>



-- 
*Henry Lyford*
Manager, Partnerships & Business Development
Edmodo.com/henry